### ISSUE TITLE
mask_img is None after fit_transform NiftiLabelsMasker

### ISSUE THREAD

**Issue by cinvro** [nilearn/nilearn#1069]
I am trying to extract time series from ADHD dataset using ALL atlas:

    from nilearn import datasets
    from nilearn.input_data import NiftiLabelsMasker,NiftiMasker

    data = datasets.fetch_adhd(n_subjects=40)
    atlas = datasets.fetch_atlas_aal()
    atlas_filename = atlas.regions

    masker = NiftiLabelsMasker(labels_img= atlas_filename, standardize=True,resampling_target='labels')
    adhd_0 = masker.fit_transform(data.func[0], confounds=data.confounds[0])

    print masker.mask_img

It is strange that the mask_img is still None after I done this. Any Idea?

Also, how can I get the mask_img which tells me which voxels are kept to form the final signal generated by the makser?

**Comment by GaelVaroquaux** [comment]
You need to look at mask_img_: the object never modifies its input, but
creates a shadow variable with an underscore.

**Comment by cinvro** [comment]
When I print the masker.mask_img_, it gives me None too.

**Comment by AlexandreAbraham** [comment]
If you dont provide `mask_img` parameter, then `mask_img_` will be None. NiftiLabelsMasker stores the label atlas in `labels_img_`. The `mask_img`parameter can be used to exclude voxels outside of the Brain. Meaning that if zone région from the labels has a part inside of the Braun and one outside (as defined by mask_img) then the mean of the voxels inside the brain is averaged at transform.
